gremlin-driver/pom.xml

```diff
@@ -98,6 +98,21 @@ limitations under the License.
             <artifactId>hamcrest-all</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```

Comment:
We use mockito for mocking and would prefer to not include additional new testing dependencies. Could you please convert your test to use that framework instead?

Reply:
Powermock is required to mock the static method of `InetAddress.getAllByName`.